apps/emqx_bridge_kinesis/src/consumer/emqx_bridge_kinesis_consumer_coordinator.erl

```diff
@@ -0,0 +1,546 @@
+%%--------------------------------------------------------------------
+%% Copyright (c) 2023 EMQ Technologies Co., Ltd. All Rights Reserved.
+%%--------------------------------------------------------------------
+-module(emqx_bridge_kinesis_consumer_coordinator).
```

Comment:
need @doc to describe the coordinator protocol

Reply:
:heavy_check_mark: https://github.com/emqx/emqx/pull/11705/commits/548b3896470007d9eaa456e005af8a28b195b6b6